### PR TITLE
Rr/rali pr351 update

### PR DESCRIPTION
**Pull request by rrawther**
Add missing file with PR 351

### CODE REVIEW

File: rali/rali/source/rali_api_augmentation.cpp [hunk 1]
```diff
@@ -1823,18 +1784,16 @@ raliCopy(
...
     try	
     {	
-        if(!input || !context)	
-            THROW("Null values passed as input")	
-
-        output = context->master_graph->create_image(input->info(), is_output);	
-
-        context->master_graph->add_node<CopyNode>({input}, {output});	
+        output = context->master_graph->create_image(input->info(), is_output);
+        context->master_graph->add_node<CopyNode>({input}, {output});
```
Comment (kiritigowda) [inline rali/rali/source/rali_api_augmentation.cpp:1796]: Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones removed
==============
+ rali/rali/source/rali_api_augmentation.cpp  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304&bid=20010190)

File: rali/rali/source/rali_api_augmentation.cpp [hunk 2]
```diff
@@ -1849,23 +1808,21 @@ raliNop(
...
     try	
     {	
-        if(!input || !context)	
-            THROW("Null values passed as input")	
-
-        output = context->master_graph->create_image(input->info(), is_output);	
-
-        context->master_graph->add_node<NopNode>({input}, {output});	
+        output = context->master_graph->create_image(input->info(), is_output);
+        context->master_graph->add_node<NopNode>({input}, {output});
```
Comment (kiritigowda) [inline rali/rali/source/rali_api_augmentation.cpp:1820]: Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones removed
==============
+ rali/rali/source/rali_api_augmentation.cpp  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304&bid=20010190)

File: rali/rali/source/rali_api_augmentation.cpp (hunk 1 above)
Comment (kiritigowda) [inline rali/rali/source/rali_api_augmentation.cpp:1796]: Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304)

File: rali/rali/source/rali_api_augmentation.cpp (hunk 2 above)
Comment (kiritigowda) [inline rali/rali/source/rali_api_augmentation.cpp:1820]: Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304)

File: rali/rali/source/rali_api_augmentation.cpp (hunk 1 above)
Comment (rrawther) [reply rali/rali/source/rali_api_augmentation.cpp:1796]: Please ignore this warning

File: rali/rali/source/rali_api_augmentation.cpp (hunk 2 above)
Comment (rrawther) [reply rali/rali/source/rali_api_augmentation.cpp:1820]: please ignore this.

**Review by kiritigowda — APPROVED** [review]

**Comment by LakshmiKumar23** [comment]
@kiritigowda Tested can be merged

**Pull request merged**